src/Data/Nat/Properties.agda

```diff
@@ -2099,8 +2088,8 @@ _>″?_ = flip _<″?_
 ≤″-irrelevant : Irrelevant _≤″_
 ≤″-irrelevant {m} (less-than-or-equal eq₁)
                   (less-than-or-equal eq₂)
-  with +-cancelˡ-≡ m _ _ (trans eq₁ (sym eq₂))
-... | refl = cong less-than-or-equal (≡-irrelevant eq₁ eq₂)
+  with refl ← +-cancelˡ-≡ m _ _ (trans eq₁ (sym eq₂))
+  = cong less-than-or-equal (≡-irrelevant eq₁ eq₂)
```

Comment:
NB (cf. our discussion of v2.0 this morning)
*This* instance of the `with refl ←...` idiom *does not* typecheck if expressed in terms of `rewrite...` (!?)
Of course, the original with `... | refl = ` still works, but I'm going to leave this instance of using irrefutable `with` as it stands.